tools/plugin/README.md

```diff
@@ -66,9 +66,9 @@ The above example needs to be 48k as example pipe has no SRC/ASRC.
 Likewise
 
 ```
-arecord -Dsof:tgl-nocodec:1:default:default:48k2c16b -f dat file.wav
+arecord -Dsof:plugin:1:default:default:48k2c16b -f dat file.wav
```

Comment:
it:would:not:hurt:to:document:all:these:options:never:seen:this:before

Reply:
@plbossart it is documented in the plugin usage help message and in the same README file below